app/src/main/java/no/javazone/scheduler/ui/theme/Theme.kt

```diff
@@ -36,10 +36,11 @@ private val LightThemeColors = lightColors(
 )
 
 private val DarkThemeColors = darkColors(
-    primary = Red300,
-    primaryVariant = Red700,
+    primary = Red700,
+    primaryVariant = Red300,
     onPrimary = Color.Black,
-    secondary = Red300,
+    secondary = Red700,
```

Comment:
bør kanskje endre dette fra rødt til noe annet?  Dette fargevalget var kopiert fra Jetnews og er vel egentlig ikke noe jeg har reelt valgt

Reply:
Ja, enig - dette er bare for å få det til å bli riktig på darkmode